src/views/Home.css

```diff
@@ -0,0 +1,78 @@
+.home {
+	background: rgba(255, 255, 255, 0.24);
+	border-radius: 16px;
+	box-shadow: 0 4px 30px rgba(0, 0, 0, 0.1);
+	backdrop-filter: blur(4.7px);
+	-webkit-backdrop-filter: blur(4.7px);
+	border: 1px solid rgba(255, 255, 255, 0.3);
+	@apply rounded-2xl shadow-2xl px-4 md:px-24 text-black mt-8;
+}
+#list {
+	@apply mt-8 flex flex-col justify-center items-center;
+}
+.input {
+	@apply border border-blue-700 text-black bg-white p-3 rounded-xl;
+	width: 70%;
+}
+.btn {
+	background-color: #113767;
+	@apply text-white p-3 rounded-xl shadow-md mb-8;
+}
+:hover .btn {
+	background-color: #38598b;
+}
+.exit-btn {
+	background-color: yellow;
+	color: black;
+	@apply p-3 rounded-xl shadow-md;
```

Comment:
nice use of the `@apply` directive to make your code more readable  👍

Reply:
thank u